streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -384,6 +413,23 @@ public void punctuate(long streamTime) {
     }
 
     /**
+     * A processor that forwards messages with modified values (without custom timestamp information) to each child, if
+     * the value is in ".*@[0-9]+" format.
+     */
+    protected static class ValueTimestampProcessor extends AbstractProcessor<String, String> {
```

Comment:
It isn't clear to me why we need this class. We can test the timestamp extraction without it, right?